sympy/tensor/array/arrayop.py

```diff
@@ -208,3 +208,38 @@ def derive_by_array(expr, dx):
             return ImmutableDenseNDimArray([expr.diff(i) for i in dx], dx.shape)
         else:
             return diff(expr, dx)
+
+
+def permutedims(expr, perm):
+    """
+    Permutes the indices of an array.
+
+    Parameter specifies the permutation of the indices.
+    """
+    if not isinstance(expr, NDimArray):
+        raise TypeError("expression has to be an N-dim array")
+
+    perm = tuple(perm)
```

Comment:
Uh, apparently it's Mathematica that uses the inverse as the definition for a permutation. Julia and NumPy follow the same convention as SymPy. I was testing on Mathematica and didn't realize that.